phoenix-core-client/src/main/java/org/apache/phoenix/util/PhoenixRuntime.java

```diff
@@ -539,6 +541,18 @@ public static PTable getTable(Connection conn, @Nullable String tenantId, String
         return pconn.getTable(tenantId, fullTableName, timestamp);
     }
 
+    /**
+     * Returns the most recent PTable fetched from the server without updating the CQSI cache.
+     */
+    public static PTable getTableFromServerNoCache(Connection conn, byte[] schemaName,
```

Comment:
The API signature is not consistent with other getTable APIs in this class.

Reply:
Maybe then you shouldn't add an API in PhoenixRuntime and directly use connection.